README.md

```diff
@@ -1,176 +1,61 @@
+Code examples for each language's SDK can be found within the following sub-directories:
+
+| SDK        | folder                                | SDK version | SDK status  |
+|------------|---------------------------------------|-------------|-------------|
+| .NET       | [dotnetv3/](dotnetv3)                 | 3.5+        | ![[]](https://img.shields.io/badge/-GA-blue)           |
+| .NET       | [dotnet/](dotnet)                     | <3.5        | ![[]](https://img.shields.io/badge/-deprecated-red)  |
+| C++        | [cpp/](cpp)                           | 1           | ![[]](https://img.shields.io/badge/-GA-blue)          |
+| Go         | [gov2/](gov2)                         | 2           | ![[]](https://img.shields.io/badge/-GA-blue)          |
+| Go         | [go/](go)                             | 1           | ![[]](https://img.shields.io/badge/-deprecated-red)  |
+| Java       | [javav2/]()                           | 2           | ![[]](https://img.shields.io/badge/-GA-blue)          |
+| Java       | [java/](java)                         | 1           | ![[]](https://img.shields.io/badge/-deprecated-red)  |
+| JavaScript | [javascriptv3/](javascriptv3)         | 3           | ![[]](https://img.shields.io/badge/-GA-blue)          |
+| JavaScript | [javascript/](javascript)             | 2           | ![[]](https://img.shields.io/badge/-deprecated-red)  |
+| Kotlin     | [kotlin/](kotlin)                     |             | ![[]](https://img.shields.io/badge/-preview-brightgreen)       |
+| PHP        | [php/](php)                           | 3           | ![[]](https://img.shields.io/badge/-GA-blue)          |
+| Python     | [python/](python)                     | 3           | ![[]](https://img.shields.io/badge/-GA-blue)          |
+| Ruby       | [ruby/](ruby)                         | 3           | ![[]](https://img.shields.io/badge/-GA-blue)          |
+| Rust       | [rust_dev_preview/](rust_dev_preview) |             | ![[]](https://img.shields.io/badge/-preview-brightgreen) |
+| Swift      | [swift/](swift)                       |             | ![[]](https://img.shields.io/badge/-preview-brightgreen)     |
+
+Within each directory, you will find SDK-specific instructions for understanding and invoking example code.
+
+## Invoking example code
+To invoke this example code, you must have an AWS account. For more information about creating an account, see [AWS Free Tier](https://aws.amazon.com/free/). 
+
+You must also have AWS credentials configured. For steps on using the CLI to configure credentials, see [CLI Configuration basics](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-quickstart.html)
+
+## ⚠️ Usage Disclaimer
+These code examples interact with services that may incur charges to your AWS account. For more information, see [AWS Pricing](https://aws.amazon.com/pricing/).
+
+Additionally, example code may theoretically modify or delete existing AWS resources. As a matter of due diligence:
```

Comment:
may theoretically --> might
As a matter of due diligence: --> As a matter of due diligence, do the following: